pkg/secrets/secrets.go

```diff
@@ -45,17 +40,17 @@ func ReplaceSecretsInPath(archiveDir string) error {
 	case "sealedsecrets":
 		return replaceSecretsWithSealedSecrets(archiveDir, secret.Data)
 	default:
-		return errors.Errorf("unkknown secret type %q", secretType)
+		return errors.Errorf("unknown secret type %q", secretType)
 	}
 }
 
-func getSecretsInPath(archiveDir string) ([]string, error) {
-	secretPaths := []string{}
+func findPathsWithSecrets(archiveDir string) ([]string, error) {
+	var paths []string
 	decode := scheme.Codecs.UniversalDeserializer().Decode
 
 	err := filepath.Walk(archiveDir, func(path string, info os.FileInfo, err error) error {
 		if err != nil {
-			return err
+			return errors.Wrap(err, "could not walk through the archive directory")
```

Comment:
this message should be moved to line 84 https://github.com/replicatedhq/kots/pull/2785/files#diff-29941b86f5d6066d562f03783c8316b6c4f13dfddcdc57c320fea3d53a6635b4R83
```suggestion
			return err
```

Reply:
good call. fixed and pushed